### PR TITLE
Use Marathon 0.15.3

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

**Comment by aquamatthias** [comment]
@jsancio please review, since I am not sure about the format changes.

**Comment by jsancio** [comment]
Looking.

File: repo/packages/M/marathon/1/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-small.png",
+    "icon-medium" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-medium.png",
+    "icon-large" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-large.png"
+  },
+  "assets" : {
+    "container" : {
+      "docker" : {
+        "ebbc8372a1c4" : "mesosphere/marathon:v0.15.3"
```
Comment (jsancio) [inline repo/packages/M/marathon/1/resource.json]: We should reference this in `marathon.json.mustache`. Checkout out:

https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/M/marathon/0/marathon.json.mustache#L26-L32

**Comment by jsancio** [comment]
Ping me and I can test this against my cluster.

**Comment by aquamatthias** [comment]
@jsancio that is needed ;). added the docker section. Please test.

**Comment by jsancio** [comment]
@aquamatthias, @jrx tested this and it looks good. Do you want to rebase and merge it? Thanks!

**Pull request merged**